VastClient/VastClient/VastTracker.swift

```diff
@@ -9,20 +9,32 @@
 import Foundation
 
 public protocol VastTrackerDelegate {
-    func adBreakStart(_ id: String, _ vastModel: VastModel)
-    func adStart(_ id: String, _ ad: VastAd)
-    func adFirstQuartile(_ id: String, _ ad: VastAd)
-    func adMidpoint(_ id: String, _ ad: VastAd)
-    func adThirdQuartile(_ id: String, _ ad: VastAd)
-    func adComplete(_ id: String, _ ad: VastAd)
-    func adBreakComplete(_ id: String, _ vastModel: VastModel)
+    func adBreakStart(vastTracker: VastTracker, totalAds: Int)
+    func adStart(vastTracker: VastTracker, ad: VastAd)
+    func adFirstQuartile(vastTracker: VastTracker, ad: VastAd)
+    func adMidpoint(vastTracker: VastTracker, ad: VastAd)
+    func adThirdQuartile(vastTracker: VastTracker, ad: VastAd)
+    func adComplete(vastTracker: VastTracker, ad: VastAd)
+    func adBreakComplete(vastTracker: VastTracker, vastModel: VastModel)
```

Comment:
with the request to make the vastModel public below this argument can be removed

Reply:
It is common to send the caller along for delegate methods in case the delegate handles multiple sources. If you don’t the delegate can’t know from where the delegate call came.